memos/umbrel-app.yml

```diff
@@ -0,0 +1,53 @@
+manifestVersion: 1
+id: memos
+category: files
+name: Memos
+version: "0.21"
+tagline: Easily capture and share your great thoughts.
+description: >-
+  A privacy-first, lightweight note-taking service.
+  Easily capture and share your great thoughts.
+
+  Open source and free forever. Embrace a future where creativity knows no boundaries with our open-source solution – free today, tomorrow, and always.
+
+  Self-hosting with Docker in just seconds. Enjoy the flexibility, scalability, and ease of setup that Docker provides, allowing you to have full control over your data and privacy.
+
+  Pure text with added Markdown support. Say goodbye to the overwhelming mental burden of rich formatting and embrace a minimalist approach.
+
+  Customize and share your notes effortlessly. With our intuitive sharing features, you can easily collaborate and distribute your notes with others.
+
+  RESTful API for third-party services. Embrace the power of integration and unleash new possibilities with our RESTful API support.
+releaseNotes: >-
+  chore\: update i18n with Weblate by @michaelien in #3079
+  chore\: fix typo by @thirdkeyword in #3080
+  chore\: fix spanish typos by @magomzr in #3083
+  chore\: update build scripts by @lincolnthalles in #3085
+  feat\: implemented link previews (proto files) by @MehadND in #3072
+  feat\: implemented link previews (server files) by @MehadND in #3073
+  chore\: update i18n with Weblate by @boojack in #3090
+  feat\: implemented link previews (frontend files) by @MehadND in #3074
+  chore\: fix typo by @sunxunle in #3106
+  feat\: add shortcut to edit the previous memo by @Dubzer in #3122
+  removed sanitization logic by @YohannKovacs in #3147
+  chore\: update Vietnamese translation by @ntheanh201 in #3155
+  fix\: link embedded content to uid instead of name by @ntheanh201 in #3164
+  feat\: handle restore memo when is in archived memo detail page by @ntheanh201 in #3165
+  cmd\: use the built-in slices library by @stayweek in #3172
+  chore\: fix memo link by @ntheanh201 in #3173
+  Set navigation icons to not shrink by @imikod in #3186
+  improve localization by @imikod in #3190
```

Comment:
We can remove release notes for the initial release on the app store:

`releaseNotes: ""`